### ISSUE TITLE
`prefer-math-trun`: Add match for `parseInt(String(num), 10)`

### ISSUE THREAD

**Issue by omril1** [sindresorhus/eslint-plugin-unicorn#2514]
### Description

This should match as well since `parseInt` ignores the fraction part

### Fail

```js
const num = 37.4;
const truncated = parseInt(String(num)), 10);
```

### Pass

```js
const um = 37.4;
const truncated = Math.trunc(num);
```

**Comment by axetroy** [comment]
Personally, I prefer `parseInt` over `Math.trunc`

`parseInt` is more readable

**Comment by sindresorhus** [comment]
Accepted
